flow-typed/npm/classnames_v2.x.x.js

```diff
@@ -1,8 +1,6 @@
 // flow-typed signature: 04e310e8c98cdb5de377193da621970b
 // flow-typed version: 7fd0a6404e/classnames_v2.x.x/flow_>=v0.25.x
 
-// @flow
-
```

Comment:
probably someone added it because of the eslint error I now removed.

Reply:
Yeah, I believe I did.